Library/test-helpers/lib.sh

```diff
@@ -1715,6 +1715,254 @@ limeSubmitCommonLogs() {
 
 }
 
+true <<'=cut'
+=pod
+
+=head2 limeconCreateNetwork
+
+Create container network.
+
+    limeconCreateNetwork
+
+=over
+
+=item NETWORK_IP
+
+IP of container network.
+
+=item NAME_NETWORK
+
+Name of container network.
+
+=back
+
+Returns 0.
+
+=cut
+
+limeconCreateNetwork() {
+
+    local NETWORK_IP=$1
+    local NAME_NETWORK=$2
+
+    if [ -z "${NAME_NETWORK}" ] || [ -z "${NETWORK_IP}" ]; then
+        echo "Name of network or network IP is not specified!"
+        return
+    fi
+
+    podman network create --subnet=$NETWORK_IP $NAME_NETWORK
+    echo "Network name: ${NAME_NETWORK} , ip:${NETWORK_IP}"
+
+}
+
+true <<'=cut'
+=pod
+
+=head2 limeconDeleteNetwork
+
+Delete container network.
+
+    limeconDeleteNetwork
+
+=over
+
+=item NAME_NETWORK
+
+Name of the container network.
+
+=back
+
+Returns 0.
+
+=cut
+
+limeconDeleteNetwork() {
+
+    local NAME_NETWORK=$1
+
+    podman network inspect $NAME_NETWORK
```

Comment:
`inspect` doesn't do a delete, just print details.. Btw, that `podman network inspect` command might be good to add to the create function above since creation doesn't produce any nice output it seems

Reply:
Oh my bad, thank you for double check, I forgot to rewrite fix into PR which I had in VM.